### PR TITLE
docs: update `properties.mdx` for icons

### PR DESCRIPTION
**Pull request by nmerget**
## Proposed changes

We show all icons in `properties.mdx` as string literal which was very long. We should replace it with a link to the icon overview

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [x] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: showcases/patternhub/scripts/get-properties-file.js [hunk 1]
```diff
@@ -60,14 +60,23 @@ const getPropertiesFile = ({ displayName, description, props }) => {
...
-						'<T>',
-						''
-					)}</code></pre>`
-				: ''
-		} |\n`;
+
+		if (
+			propertyKey === 'icon' ||
+			propertyKey === 'iconAfter' ||
+			propertyKey === 'messageIcon'
```
Comment (mfranzke) [inline showcases/patternhub/scripts/get-properties-file.js:67]:
```suggestion
			['icon', 'iconAfter', 'messageIcon'].includes(propertyKey)
```

**Comment by mfranzke** [comment]
@nmerget it's not 100% clear to me what this PR is actually about. The icons don't include a properties page, so I assume that this is about the other components properties pages. In which the properties pages are currently failing: https://db-ui.github.io/mono/review/docs-icon-properties/components/button/properties

**Comment by nmerget** [comment]
> @nmerget it's not 100% clear to me what this PR is actually about. The icons don't include a properties page, so I assume that this is about the other components properties pages. In which the properties pages are currently failing: https://db-ui.github.io/mono/review/docs-icon-properties/components/button/properties

It's not about the failing properties, it's more about the content inside. Currently we show all literal strings for icons which is way to big and confusing to read:
image

With a simple link to icon overview we provide a better alternative

**Comment by mfranzke** [comment]
> > @nmerget it's not 100% clear to me what this PR is actually about. The icons don't include a properties page, so I assume that this is about the other components properties pages. In which the properties pages are currently failing: https://db-ui.github.io/mono/review/docs-icon-properties/components/button/properties
> 
> It's not about the failing properties, it's more about the content inside. Currently we show all literal strings for icons which is way to big and confusing to read: image
> 
> With a simple link to icon overview we provide a better alternative

grazie mille, now I got it.

**Review by mfranzke — APPROVED** [review]

**Pull request merged**